### PR TITLE
Fix: Scoreboard cut off when player count is high

### PR DESCRIPTION
**Pull request by mzegar**

### CODE REVIEW

File: code/ui/generalhud/scoreboard/Scoreboard.cs [hunk 1]
```diff
@@ -70,13 +70,11 @@ private void OnPlayerSpawned(TTTPlayer player)
         private void AddPlayer(PlayerScore.Entry entry)
         {
             ScoreboardGroup scoreboardGroup = GetScoreboardGroup(entry);
-            ScoreboardEntry scoreboardEntry = scoreboardGroup.AddEntry(entry);
-
             scoreboardGroup.GroupMembers++;
+            scoreboardGroup.UpdateLabel();
 
-            _entries.Add(entry.Id, scoreboardEntry);
+            _entries.Add(entry.Id, scoreboardGroup.AddEntry(entry));
```
Comment (mzegar) [inline code/ui/generalhud/scoreboard/Scoreboard.cs:76]: Just reordering for readability, I did some testing and it seemed like it had no issues.

File: code/ui/generalhud/scoreboard/Scoreboard.scss [hunk 2]
```diff
@@ -14,6 +12,8 @@ Scoreboard {
     display: flex;
     flex-direction: column;
     background-color: #000c;
+    align-self: center;
```
Comment (Alf21) [inline code/ui/generalhud/scoreboard/Scoreboard.scss:15]: Is this supported?

**Review by Alf21 — APPROVED** [review]
I think it is caused by the transform / matrix issue I experienced and reported as well.

File: code/ui/generalhud/scoreboard/Scoreboard.scss (hunk 2 above)
Comment (mzegar) [reply code/ui/generalhud/scoreboard/Scoreboard.scss:15]: ye, worked within game.

**Pull request merged**